src/ClusterManager/job_manager.py

```diff
@@ -333,12 +333,24 @@ def UpdateJobStatus(redis_conn,
         dataHandler = DataHandler()
     else:
         dataHandler = dataHandlerOri
+    jobParams = json.loads(
+        base64.b64decode(job["jobParams"].encode("utf-8")).decode("utf-8"))
 
     result, details = launcher.get_job_status(job["jobId"])
     logger.info("++++++++ Job status: {} {}".format(job["jobId"], result))
 
+    jobPath, workPath, dataPath = GetStoragePath(jobParams["jobPath"],
+                                                 jobParams["workPath"],
+                                                 jobParams["dataPath"])
+    localJobPath = os.path.join(config["storage-mount-path"], jobPath)
+    logPath = os.path.join(localJobPath, "logs/joblog.txt")
+
+    if "userId" not in jobParams:
+        jobParams["userId"] = "0"
+
     if result == "Succeeded":
-        joblog_manager.extract_job_log(job["jobId"])
+        joblog_manager.extract_job_log(job["jobId"], logPath,
```

Comment:
This means writing to NFS upon success/failure even if elasticsearch is deployed. @xudifsd do you think it's ok?

Reply:
This will be temporary, right?

Reply:
No, this will save to `/dlwsdata/work/<alias>/jobs/...../logs/joblog.txt`

Reply:
I know, I meant this solution will be temporary, right? Then I'm fine with this.